src/main/java/com/github/wovnio/wovnjava/QueryPatternHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.regex.Pattern;
+
+class QueryPatternHandler extends PatternHandler {
+    static final String QueryPatternRegex = "(?:(?:\\?.*&)|\\?)wovn=([^&]+)(?:&|$)";
```

Comment:
You might want to review naming convention either here or at https://github.com/WOVNio/wovnjava/pull/104/files#diff-a28a80824ff1f2d1b9b046bfe472ad20R6

Reply:
Renamed consistently now.

Reply:
thanks